presto-docs/src/main/sphinx/develop/event-listener.rst

```diff
@@ -0,0 +1,48 @@
+==============
+Event Listener
+==============
+
+Presto supports custom event listeners that are invoked when these events
+occur:
+
+* A query is created
+* A query completes or fails
+* A split completes or fails
+
+This functionality allows developers to build custom logging, debugging and
+performance analysis plugins. Only a single event listener plugin can be
```

Comment:
Only a single event listener plugin may be activate at a time in a Presto cluster.